src/routes/(app)/api/search/meilisearch.ts

```diff
@@ -0,0 +1,20 @@
+import { Meilisearch } from "meilisearch";
+import { env } from "$env/dynamic/private";
+
+export const meilisearch = new Meilisearch({
+  host: env.MEILISEARCH_HOST,
+  apiKey: env.MEILISEARCH_MASTER_KEY,
+  requestConfig: {
+    headers: {
+      Authorization: `Bearer ${env.MEILISEARCH_MASTER_KEY}`,
+    },
+  },
+});
+
+await Promise.all([
+  meilisearch.createIndex("positions"),
+  meilisearch.createIndex("members"),
+  meilisearch.createIndex("songs"),
+  meilisearch.createIndex("articles"),
+  meilisearch.createIndex("events"),
+]);
```

Comment:
Like we spotted earlier, this should not be called at the top level. It breaks CI and I guess it should only really be called once?